trackbar_base.h

```diff
@@ -41,8 +40,10 @@ class NOVTABLE TrackbarCallback {
      *
      * \return                    whether you processed the key state change
      */
-    virtual bool on_key(WPARAM wp, LPARAM lp) { return false; }
-};
+    virtual bool on_key(WPARAM wp, LPARAM lp){return false;
```

Comment:
This also isn't right...

Reply:
Resolved – `NOVTABLE` needed to be added to `AttributeMacros` in `.clang-format`.